src/jit/lsraarm.cpp

```diff
@@ -454,103 +489,22 @@ void Lowering::TreeNodeInfoInitCall(GenTreeCall* call)
             continue;
         }
 
-        var_types argType    = argNode->TypeGet();
-        bool      argIsFloat = varTypeIsFloating(argType);
-        callHasFloatRegArgs |= argIsFloat;
-
-        regNumber argReg = curArgTabEntry->regNum;
-        // We will setup argMask to the set of all registers that compose this argument
-        regMaskTP argMask = 0;
```

Comment:
`argMask` was only ever computed per reg argument. We still compute `argMask`, but this computation is now encapsulated in `TreeNodeInfoInitPutArgReg`.